other/cmr-exchange/service-bridge/test/cmr/opendap/tests/integration/app/ous.clj

```diff
@@ -13,12 +13,40 @@
   * https://en.wikipedia.org/wiki/Software_testing#Integration_testing"
   (:require
    [clojure.test :refer :all]
+   [cmr.http.kit.request :as request]
    [cmr.http.kit.response :as response]
    [cmr.opendap.testing.system :as test-system]
+   [cmr.opendap.testing.util :as util]
    [org.httpkit.client :as httpc]))
 
 (use-fixtures :once test-system/with-system)
 
+(deftest cmr-hits-test
+  (let [collection-id "C1200267318-HMR_TME"
+        options (-> {}
+                    (request/add-token-header (util/get-sit-token))
+                    (util/override-api-version-header "v2.1"))]
```

Comment:
`v2.1` is not the latest api version. We don't need to force the version be `v2.1`